mautic/mautic
# Anniversary segment filter

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | ✓
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR adds the ability to segment contacts whose birthday is the current date through a segment filter.

![image](https://user-images.githubusercontent.com/18265735/27783732-b3a62f56-5fd7-11e7-9feb-b3cd5de361f7.png)

#### Steps to test this PR:
1. Create a date custom field to hold the contact's birthday
2. Create a contact whose birthday is the current date but not the current year
3. Create a segment with a filter on that field with the _equals_ operator and _birthday_ as the value (or whatever the value of the translation key `mautic.lead.list.birthday` is in your locale)
4. Refresh the segments
5. The contact should now be in the segment
